lib/lotus/helpers/html_helper.rb

```diff
@@ -184,8 +193,13 @@ module HtmlHelper
       #
       # @see Lotus::Helpers::HtmlHelper
       # @see Lotus::Helpers::HtmlHelper::HtmlBuilder
-      def html
-        HtmlBuilder.new
+      # @see Lotus::Helpers::HtmlHelper::HtmlFragment
+      def html(&blk)
+        if block_given?
+          HtmlBuilder.new.fragment(&blk)
```

Comment:
@jodosha I'm afraid not, as `yield` is supposed to use the context where the block was defined.

I actually tried to executing this in the test with `yield`

```ruby
html do
  div 'Lotus'
end

# => NoMethodError: undefined method `div' for #<HtmlView:0x007fe5422fdaa8>
```